hypervisor/arch/x86/cpu.c

```diff
@@ -482,6 +482,9 @@ static void bsp_boot_post(void)
 	interrupt_init(BOOT_CPU_ID);
 
 	timer_init();
+#ifdef HV_DEBUG
```

Comment:
we actually could remove HV_DEBUG here, you can do it like what we handle dump_exception.